test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -342,6 +356,7 @@ describe('v1.0.0 to v2.0.0', function () {
 
                 it('should correctly infer a noauth type from `currentHelper`', function (done) {
                     var source = {
+                        id: '78935144-80d0-4018-bfdd-5f321b3e4674',
```

Comment:
Any particular reason ids were added to these tests? Would they work if the ids were not supplied?

Reply:
It would still work, but the assertions would have to be altered to work dynamically for the generated id. This allows keeping the assertion straightforward, and actually ensures that the id is correctly retained.